prober/icmp.go

```diff
@@ -80,6 +80,11 @@ func ProbeICMP(ctx context.Context, target string, module config.Module, registr
 	}
 
 	registry.MustRegister(durationGaugeVec)
+	ttlGauge := prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "probe_icmp_reply_ttl_total",
```

Comment:
Gauges shouldn't end in total, that's for counters.

Reply:
OK. I renamed it to probe_icmp_reply_hop_limit as even IPv6 header renamed ttl to hop_limit